tests/data/author/governed_folders/template_folder_with_template_type/network.md

```diff
@@ -0,0 +1,16 @@
+---
+authors: tmp
+owner: tmp
+valid:
+  from:
+  to:
+x-trestle-template-type: network
+---
```

Comment:
x-trestle-template-type has to be removed from the template

Reply:
done